packages/connect-thegraph-tokens/subgraph/manifest/data/mainnet-staging.json

```diff
@@ -3,16 +3,22 @@
   "dataSources": {
     "Organizations": [
       {
-        "name": "PieDAO",
-        "address": "0x0c188B183FF758500D1D18B432313d10e9F6b8a4",
-        "startBlock": "9593674"
+        "name": "DeepDAO",
```

Comment:
Here's where we set the parameters to test the stranger case in fast staging mode.